src/main/java/htsjdk/variant/vcf/VCF3Codec.java

```diff
@@ -129,8 +108,20 @@ else if (line.startsWith(VCFHeader.HEADER_INDICATOR)) {
         return fFields;
     }
 
+    /**
+     * Handle reporting of duplicate filter IDs
+     * @param duplicateFilterMessage
+     */
     @Override
-    public boolean canDecode(final String potentialInput) {
-        return canDecodeFile(potentialInput, VCF3_MAGIC_HEADER);
+    protected void reportDuplicateFilterIDs(final String duplicateFilterMessage) {
+        // no-op since this codec's parseFilters method doesn't check for them
     }
+
+    /**
+     * Handle report of duplicate info field values
+     * @param key
+     * @param infoLine
+     */
+    public void reportDuplicateInfoKeyValue(final String key, final String infoLine) {}
```

Comment:
Yes, there should be a comment here. I made this a no-op because that was the previous behavior.